### ISSUE TITLE
Can't create the user for jenkins using script conf-jenkins-user.sh

### ISSUE THREAD

**Issue by cmoulliard** [christian-posta/ceposta-devops-ose#5]
When we run the script `conf-jenkins-user.sh`, we get this error 
```
dabou:~/Temp/poc-demos-test/ceposta-devops-ose/gerrit-docker$ ./conf-jenkins-user.sh 192.168.59.103 32772
ssh_exchange_identification: Connection closed by remote host
```

```
dabou:~/Temp/poc-demos-test/ceposta-devops-ose/bootstrap$ ./print-docker.sh
Docker host name is 192.168.59.103
Gerrit:  http://192.168.59.103:32772/
Gitlab:  http://192.168.59.103:32770/
Jenkins: http://192.168.59.103:32776/
Nexus:   http://192.168.59.103:32774/nexus
```

**Comment by jstrachan** [comment]
its maybe more useful to get the kubernetes apps for jenkins / nexus et al working in the fabric8 apps?
https://github.com/fabric8io/quickstarts/tree/master/apps

then there'll be no IP issues as things wire together with kubernetes services nicely

**Comment by christian-posta** [comment]
Yes @jstrachan i think that is the next logical step for this demo; not to use hand crafted/orchestrated docker containers ... @cmoulliard the issue you have is you're using the docker container IP directly; you should use the docker host IP and ports

**Comment by jstrachan** [comment]
BTW I've got jenkins, nexus, gerrit and sonaqube running on master of the fabric8 apps. Am using Nexus and Jenkins right now on our internal OS1 cloud...

**Comment by jstrachan** [comment]
along with gogs (git hosting), taiga (issue tracker / kanban board), letschat / hubot for [chat]
(http://fabric8.io/guide/chat.html) etc.

**Comment by christian-posta** [comment]
@jstrachan ah, very nice. will try to merge some of my ideas from this repo to fabric8v2 in the next couple of weeks

**Comment by jstrachan** [comment]
@christian-posta awesome! BTW I should have said - the apps are all here:
https://github.com/fabric8io/quickstarts/tree/master/apps

and they appear with 1 click install/run in the [fabric8 console](http://fabric8.io/guide/console.html)
